docs/src/pages/components/autocomplete/CountrySelect.js

```diff
@@ -25,10 +15,14 @@ export default function CountrySelect() {
       renderOption={(props, option) => (
         <Box
           component="li"
-          sx={{ fontSize: 15, '& > span': { mr: '10px', fontSize: 18 } }}
+          sx={{ fontSize: 15, '& > img': { mr: '10px' } }}
           {...props}
         >
-          <span>{countryToFlag(option.code)}</span>
+          <img
+            loading="lazy"
+            src={`https://www.countryflags.io/${option.code}/shiny/16.png`}
```

Comment:
- The image needs to handle high PPI screens
- The image needs to reserve space until loaded not to create a layout shift
- Are we sure about the shiny theme?

Reply:
Fixed all, used https://flagpedia.net/, it's more serious than https://www.countryflags.io.